compiler-core/src/language_server/tests/hover.rs

```diff
@@ -14,51 +14,51 @@ fn hover(tester: TestProject<'_>, position: Position) -> Option<Hover> {
     })
 }
 
-struct PositionFinder {
+pub struct PositionFinder {
```

Comment:
If this is used by other tests let's move it out of the hover specific tests. Could be in `tests.rs`